src/data/societies.ts

```diff
@@ -751,6 +751,33 @@ export const SOCIETIES_ALL: ISociety[] = [
     links: {
       fb: 'https://www.facebook.com/kgpspeedcubers/',
     }
+  },
+  {
+    symbol: 'Sa',
+    name: "Spark4AI",
+    year: 2023,
+    size: 'L',
+    tsg_recognized: false,
+    description: "SPARK4AI is a vibrant community of AI enthusiasts united by a common passion for harnessing the transformative power of Artificial Intelligence.",
+    block: 'TO',
+    links: {
+      insta: 'https://www.instagram.com/spark4ai.iitkgp/',
+      linkedin: 'https://www.linkedin.com/company/spark4ai/'
+    }
+  },
+  {
+    symbol: 'Pg',
+    name: "Public Policy and Governance Society",
+    year: 2023,
+    size: 'L',
+    tsg_recognized: false,
+    description: "Public Policy and Governance Society, IIT Kharagpur serves as a dynamic platform for impassioned students to rigorously examine, deliberate upon, and actively engage with current public Policy and governance issues.",
```

Comment:
A more neutral and descriptive sentence would be better here. This feels very GPT'd.